README.md

```diff
@@ -27,43 +27,65 @@ Our Task Tracker application defines four *Application Roles*:
 - Approver: Has the ability to change the status of tasks.
 - Observer: Only has the ability to view tasks and their statuses.
 
-These application roles are defined in the [Azure Management Portal](https://manage.windowsazure.com/) on the application registration page.  When a user signs into the application, AAD emits a Role Claim for each role that has been granted based on the user and their group membership.  Assignment of users and groups to roles can be done through the portal's UI, or programatically using the [AAD Graph API](http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx).  In this sample, application role management is done through the Azure Portal.
+These application roles are defined in the [Azure Management Portal](https://manage.windowsazure.com/) on the application registration page.  When a user signs into the application, AAD emits a Role Claim for each role that has been granted based on the user and their group membership.  Assignment of users and groups to roles can be done through the portal's UI, or programmatically using the [AAD Graph API](http://msdn.microsoft.com/en-us/library/azure/hh974476.aspx).  In this sample, application role management is done through the Azure Portal.
 
 Using RBAC with Application Roles and Role Claims, this application securely enforces authorization policies with minimal effort on the part of the developer.
 
 NOTE: Role claims are not currently emitted in SAML tokens, only JWTs (see issue #1).
 
 NOTE: Role claims are not currently emitted for guest users in a tenant (see issue #2).
 
-## How To Run The Sample as a MultiTenant App
+## How to build the sample?

```

Comment:
Re-structuring the document to have common steps upfront, and then the 2 cases (multi-tenant, and single-tenant)